MetadataExtractor/IO/IndexedCapturingReader.cs

```diff
@@ -151,8 +160,7 @@ public override byte[] GetBytes(int index, int count)
             var toIndex = 0;
             while (remaining != 0)
             {
-                var fromChunkIndex = fromIndex / _chunkLength;
-                var fromInnerIndex = fromIndex % _chunkLength;
+                GetPosition(index, out int fromChunkIndex, out int fromInnerIndex);
```

Comment:
This is the problem:

```suggestion
                GetPosition(fromIndex, out int fromChunkIndex, out int fromInnerIndex);
```